spec/rubocop/cop/rspec/predicate_spec.rb

```diff
@@ -0,0 +1,20 @@
+# encoding: utf-8
+
+describe RuboCop::Cop::RSpec::Predicate, :config do
+  subject(:cop) { described_class.new(config) }
+
+  it 'finds boolean methods in expectations' do
+    inspect_source(cop, 'expect(foo.valid?).to be_truthy')
+
+    expect(cop.offenses.size).to eq(1)
+    expect(cop.highlights).to eq(['expect(foo.valid?).to be_truthy'])
+
+    message = 'Use predicate matcher `expect(foo).to be_valid`'
+    expect(cop.messages).to eq([message])
+  end
+
+  it 'autocorrects an offense' do
+    replacement = autocorrect_source(cop, ['expect(foo.valid?).to be_truthy'])
```

Comment:
Minor issue, but is there a chance the autocorrected code spec could stricter than the original, since `be_truthy` is different than `be_true`?